src/website/app/Router.js

```diff
@@ -53,14 +57,24 @@ export default function Router({ demos }: Props) {
         render={route => {
           const { componentId, exampleId } = route.match.params;
           const selectedDemo = demos[componentId];
-          const selectedExample = selectedDemo.examples.filter(
+          const selectedExample = selectedDemo.examples.find(
             example => example.id === exampleId
-          )[0];
+          );
           const chromeless = route.location.search === '?chromeless';
+          const canonicalLink = `https://mineral-ui.com/components/${selectedDemo.title.toLowerCase()}/${selectedExample.id}`;
```

Comment:
You can just define `canonicalLink` and `pageTitle` in the `pageMeta` object like you do elsewhere.  No need for the extra variables if they are not used elsewhere.

```js
const pageMeta = {
  title: `${selectedDemo.title} ${startCase(selectedExample.id)} | Mineral UI`,
  canonicalLink: `https://mineral-ui.com/components/${selectedDemo.title.toLowerCase()}/${selectedExample.id}`
};
```

Reply:
no problem. the prettier formatting was hurting my eyes, so I pulled it out, but it's not a big deal.